packages/config/config/devices/0x0178/9125051.json

```diff
@@ -0,0 +1,81 @@
+// Nexia Home Intelligence 9125051
+// VeriLock Translator
+{
+	"manufacturer": "Nexia Home Intelligence",
+	"manufacturerId": "0x0178",
+	"label": "9125051",
+	"description": "VeriLock Translator",
+	"devices": [
+		{
+			"productType": "0x5a44",
+			"productId": "0x414e",
+			"zwaveAllianceId": 1173
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Buzzer Volume",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 100,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Discovery Mode",
+			"description": "Start, stop, or get the results of discovery mode. Retrieve the configuration parameter to obtain the results: 0=not enabled, 1=enabled, 2=timed out, 3=failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Enable",
+					"value": 1
+				},
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
```

Comment:
I think the additional values mentioned in the description should have an option (maybe with a `(readonly)` in the label)